burr/integrations/persisters/b_mongodb.py

```diff
@@ -0,0 +1,74 @@
+from pymongo import MongoClient
+import datetime
+import json
+import logging
+from typing import Literal, Optional
+from burr.core import persistence, state
+from datetime import datetime, timezone
+
+logger = logging.getLogger(__name__)
+
+class MongoDBPersister(persistence.BaseStatePersister):
+    """A class used to represent a MongoDB Persister."""
```

Comment:
Could you show some example code here on how to use it -- much like the Postgresql one https://github.com/DAGWorks-Inc/burr/blob/main/burr/integrations/persisters/postgresql.py#L18-L38 (I need to update the Redis one to be consistent here).

Reply:
Sure, Please Check Updated PR. I have done suggested changes.
I have added reference to the class in docs/reference/persister.rst.

Reply:
thanks added comment to change what you put thanks.